deployer/config_validation.py

```diff
@@ -152,3 +152,47 @@ def validate_support_config(cluster_name):
                 sys.exit(1)
     else:
         print_colour(f"No support defined for {cluster_name}. Nothing to validate!")
+
+
+def validate_authenticator_config(cluster_name, hub_name):
+    """
+    Validates the provided authenticator config for each hub of a specific cluster.
```

Comment:
It seems like this is primarily being used to check whether `auth0` has been disabled if another authenticator is being used, is that right? If so, I think we could make the function description or name more specific. Though if we intend to expand the kind of validation we do in the future then it might be fine.